packages/discord.js/typings/index.d.ts

```diff
@@ -4733,14 +4738,14 @@ export type MessageChannelComponentCollectorOptions<T extends MessageComponentIn
 export interface MessageEditOptions {
   attachments?: MessageAttachment[];
   content?: string | null;
-  embeds?: (Embed | APIEmbed)[] | null;
+  embeds?: (JSONEncodable<APIEmbed> | APIEmbed)[] | null;
   files?: (FileOptions | BufferResolvable | Stream | MessageAttachment)[];
   flags?: BitFieldResolvable<MessageFlagsString, number>;
   allowedMentions?: MessageMentionOptions;
   components?: (
     | JSONEncodable<APIActionRowComponent<APIMessageActionRowComponent>>
-    | ActionRow<MessageActionRowComponent>
-    | (Required<BaseComponentData> & ActionRowData<MessageActionRowComponentData | MessageActionRowComponent>)
+    | ActionRowBuilder<MessageActionRowComponentBuilder>
```

Comment:
this is already an accepted type in the line right above, what

Reply:
Then `ActionRowBuilder<MessageActionRowComponentBuilder>` can be removed from this union and the other one since it's already assignable to `JSONEncodable<APIActionRowComponent<APIMessageActionRowComponent>>`

Reply:
seems like you're right, will remove that then